synfig-core/src/synfig/filecontainerzip.cpp

```diff
@@ -343,8 +343,7 @@ void FileContainerZip::read_history(std::list<HistoryRecord> &list, FILE *f, fil
 	bool found = false;
 	HistoryRecord history_record;
 
-	for(int i = read_size - sizeof(EndOfCentralDirectory); i >= 0; i--)
-	{
+	for (int i = read_size - sizeof(EndOfCentralDirectory); i >= 0; i--) {
```

Comment:
Postfix seems fine to me logically unless the rh value is needed immediately.

Below you changed all to prefixes. Here I guess you missed the `i++` :P

Reply:
Some "good practices" guides recommend always prefer pre-increment operator in `for` loops. The reason is to not make mistakes and not forget to use it when is not a primitive type.

Reply:
I saw some Internet post about pre increment for user data types saving memory. Never realized post might make non-primitive inefficient dude to temp obj creation.

Your PR was kinda a good CS revision for me LOL
Glad I reviewed and learned something along the way 😅